flash_examples/pointcloud_segmentation.py

```diff
@@ -0,0 +1,45 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import flash
+from flash.core.data.utils import download_data
+from flash.pointcloud import launch_app, PointCloudSegmentation, PointCloudSegmentationData
+
+# 1. Create the DataModule
+# Dataset Credit: http://www.semantic-kitti.org/
+download_data("https://pl-flash-data.s3.amazonaws.com/SemanticKittiTiny.zip", "data/")
+
+datamodule = PointCloudSegmentationData.from_folders(
+    train_folder="data/SemanticKittiTiny/train",
+    val_folder='data/SemanticKittiTiny/val',
+)
+
+# 2. Build the task
+model = PointCloudSegmentation(backbone="randlanet_semantic_kitti", num_classes=datamodule.num_classes)
+
+# 3. Create the trainer and finetune the model
+trainer = flash.Trainer(max_epochs=1, limit_train_batches=0, limit_val_batches=0, num_sanity_val_steps=0)
+trainer.fit(model, datamodule)
+
+# 4. Predict what's within a few PointClouds?
+predictions = model.predict([
+    "data/SemanticKittiTiny/predict/000000.bin",
+    "data/SemanticKittiTiny/predict/000001.bin",
+])
+
+# 5. Save the model!
+trainer.save_checkpoint("pointcloud_segmentation_model.pt")
+
+# 6. Visualize
+app = launch_app(datamodule)
+app.show_predictions(predictions)
```

Comment:
We should move this to a `flash_examples/visualize` along with the fiftyone examples and one showing the data fetcher stuff. Can do this in a separate PR though

Reply:
Worth to brainstorm a little bit about this one.